scripts/gis-material-update/automation/requirements.txt

```diff
@@ -0,0 +1,13 @@
+pandas
+gtfs_kit
+pyyaml
+matplotlib
+sqlalchemy
+psycopg2-binary
+geoalchemy2
+pyjq
```

Comment:
Are all of these requirements used? I can't find any references to some of them.

Reply:
This set of requirements was enough to run all the current targets:
```
pandas
gtfs_kit
pyyaml
sqlalchemy
psycopg2-binary
geoalchemy2
parse
```

Reply:
Change done.